libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsPlayerAttributes.kt

```diff
@@ -61,9 +61,9 @@ sealed class VoiceInstructionsPlayerAttributes {
 
         override fun configureTextToSpeech(): TextToSpeech.(Bundle) -> Unit {
             return { bundle ->
-                bundle.putString(
+                bundle.putInt(
```

Comment:
Can we add a unit test covering this logic?

Reply:
Yes, please have a look